test/python/test_composite.py

```diff
@@ -16,16 +13,20 @@
     pass
 
 from cantera.composite import _pandas
-from . import utilities
 
+from .utilities import (
+    assertNear,
+    assertArrayNear
+
+)
 
-class TestModels(utilities.CanteraTest):
+@pytest.fixture(scope='class')
+def setup_models_tests(request, load_yaml):
+    request.cls.yml_file = request.cls.test_data_path / "thermo-models.yaml"
+    request.cls.yml = load_yaml(request.cls.yml_file)
 
-    @classmethod
-    def setUpClass(cls):
-        utilities.CanteraTest.setUpClass()
-        cls.yml_file = cls.test_data_path / "thermo-models.yaml"
-        cls.yml = utilities.load_yaml(cls.yml_file)
+@pytest.mark.usefixtures('setup_models_tests')
+class TestModels():
```

Comment:
If there's no base class, you should omit the parentheses. (Here and many other cases)
```suggestion
class TestModels:
```